examples/dyn_phy.py

```diff
@@ -0,0 +1,86 @@
+import sys
+
+sys.path.append("/usr/local/serialbox/python/")
+import serialbox
+import copy
+import json
+from argparse import ArgumentParser, Namespace
+from datetime import datetime
+from typing import Any, Dict, List
+
+import numpy as np
+import yaml
+from mpi4py import MPI
+
+import fv3core
+import fv3core._config as spec
+import fv3core.testing
+import fv3core.utils.global_config as global_config
+import fv3gfs.util as util
+import gt4py
+
+from fv3gfs.physics.stencils.physics import Physics
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+fv3core.set_backend("numpy")
+fv3core.set_rebuild(False)
+fv3core.set_validate_args(False)
+global_config.set_do_halo_exchange(True)
+
+spec.set_namelist("c12_6ranks_baroclinic_dycore_microphysics/input.nml")
+
+experiment_name = yaml.safe_load(
+    open("c12_6ranks_baroclinic_dycore_microphysics/input.yml", "r",)
+)["experiment_name"]
+
+# set up of helper structures
+serializer = serialbox.Serializer(
+    serialbox.OpenModeKind.Read,
+    "c12_6ranks_baroclinic_dycore_microphysics",
```

Comment:
since this comes up at least 3 times, could be a variable 'experiment_name' or something